build.rs

```diff
@@ -38,10 +29,29 @@ fn main() {
         .unwrap();
     let mut contents: String = String::new();
     file.read_to_string(&mut contents).unwrap();
-    contents = contents.replace(
-        "extern \"C\" {",
-        "#[link(name = \"dnssd\", kind = \"raw-dylib\")]\r\nextern \"C\" {",
-    );
+    if target_os == "windows" {
+        if target_arch == "x86" {
+            contents = contents.replace("extern \"stdcall\"", "extern \"system\"");
```

Comment:
It appears that `bindgen` provides a method for overriding the ABI, is there a reason we cannot use that here?

https://github.com/rust-lang/rust-bindgen/pull/2329

Reply:
Sorry, I just do not know that.